pkg/sdn/plugin/registry.go

```diff
@@ -263,6 +263,34 @@ func (registry *Registry) GetEgressNetworkPolicies() ([]osapi.EgressNetworkPolic
 	return policyList.Items, nil
 }
 
+// Returns true to indicate status update is needed
+type UpdateNodeFunc func(node *kapi.Node) bool
```

Comment:
There is already other code in the plugin that accesses registry.kClient directly.

The registry is an artifact from back when openshift-sdn could be run as a separate process and so there was one implementation of Registry in origin that used kClient/osClient, and one in openshift-sdn that talked to etcd directly. Now it's just an unnecessary abstraction mixed with some useful utility functions. We should be moving towards dropping the Registry type and just using kClient/osClient directly (or via utility functions where useful).